src/kernel/arch/x86/pit.zig

```diff
@@ -140,6 +148,35 @@ inline fn sendCommand(cmd: u8) void {
     arch.outb(COMMAND_REGISTER, cmd);
 }
 
+///
+/// Read the current reload value of the PIT counter.
+///
+/// Arguments:
+///     counter: u16 - The counter to read from.
+///
+/// Return: u16
+///     The reload value currently in the PIT counter.
+///
+inline fn readCounter(comptime counter: u16) u16 {
```

Comment:
What is the reload value? If it's not something simple then it may be worth adding some info regarding that.

Reply:
Forgot to remove this function bc I couldn't get it working